packages/angular_devkit/build_angular/src/dev-server/index.ts

```diff
@@ -107,6 +107,12 @@ export function serveWebpackBrowser(
     // In dev server we should not have budgets because of extra libs such as socks-js
     overrides.budgets = undefined;
 
+    // When having a serve path we need to change the
+    // baseHref from '/' as otherwise resources won't be found.
+    if (options.servePath && !(rawBrowserOptions.baseHref || options.baseHref)) {
+      overrides.baseHref = '';
```

Comment:
Won't this break routing?

The serve path option is intended as an advanced option used to override the default serve path which is calculated from the deploy URL and base HREF (useful in reverse proxy scenarios, for instance).  Using the serve path option on its own, will most likely break the application in some form.

Reply:
I see, I will test it with routing, but if the servePath is indented to be used with baseHref I will remove this part and add the baseHref in the test.

Reply:
@clydin, the router seems to be working fine. Not sure if you prefer to keep the new logic for the baseHref or remove this and rely on he fact that the baseHref will be provided by the users